anomalydetection/autoencoder_lstm.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+import tensorflow as tf
+from tensorflow.keras.models import Sequential
+from tensorflow.keras.layers import Dense, LSTM, Dropout, RepeatVector, TimeDistributed
+
+from anomalydetection.detectors import BaseDetector
+from anomalydetection.features import create_dataset
+
+
+def build_model(X_train, dropout_fraction=0.2, size=128):
+    timesteps = X_train.shape[1]
+    num_features = X_train.shape[2]
+
+    model = Sequential([
+        LSTM(size, input_shape=(timesteps, num_features)),
+        Dropout(dropout_fraction),
```

Comment:
I expected another LSTM layer here with a smaller size, in order to keep only the important features of the timeseries?

Reply:
That is the repeat vector (next line). timesteps < num_features in the input (rolling window). Makes sense?

Reply:
![image](https://user-images.githubusercontent.com/33832493/112278416-196b5100-8c83-11eb-98d4-d3168f2c4b69.png)